ops-bedrock/docker-compose-testnet.yml

```diff
@@ -0,0 +1,183 @@
+version: '3.4'
+
+# This Compose file is expected to be used with the testnet-up.sh script.
+# The volumes below mount the configs generated by the script into each
+# service.
+
+volumes:
+  l1_data:
+  l2_data:
+  op_log:
+
+
+services:
+  da:
+    container_name: celestia-light-node
+    user: root
+    platform: "${PLATFORM}"
+    image: "ghcr.io/celestiaorg/celestia-node:v0.9.5"
+    command: celestia light start --core.ip https://rpc-blockspacerace.pops.one --gateway --gateway.addr da --gateway.port 26659 --p2p.network blockspacerace
+    environment:
+      - NODE_TYPE=light
+      - P2P_NETWORK=blockspacerace
+    volumes:
+      - $HOME:/home/celestia
```

Comment:
Consider mounting only the note store location, it should still work and would be better scope wise.

Reply:
if we mount `$HOME/.celestia-<nodetype>-<network>` it will look in that directory, and not find the `.celestia-<nodetype>-<network>` directory (but its contents), and will then create a new node store at `$HOME/.celestia-<nodetype>-<network>/.celestia-<nodetype>-<network>` so I think this is best for now